Makefile

```diff
@@ -251,6 +251,16 @@ lint: ## Run lint checks for all components
 	cd $(MAKEFILE_ROOT)/utilities/db-migration && make lint
 	cd $(MAKEFILE_ROOT)/utilities/init-container && make lint
 
+gosec: ## Run gosec checks for all components
+	cd $(MAKEFILE_ROOT)/backend-shared && make gosec
+	cd $(MAKEFILE_ROOT)/backend && make gosec
+	cd $(MAKEFILE_ROOT)/cluster-agent && make gosec
+	cd $(MAKEFILE_ROOT)/appstudio-controller && make gosec
+	cd $(MAKEFILE_ROOT)/tests-e2e && make gosec
+#	cd $(MAKEFILE_ROOT)/utilities/db-migration && make gosec
+#	cd $(MAKEFILE_ROOT)/utilities/init-container && make gossec
```

Comment:
Are these comments intentional? Should we check gosec for utilities as well?

Reply:
Thanks Chetan, this was mostly a placeholder so we remember to add gosec to these in the future.